demos/demos_databases_apis/graphviz/graphviz.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Graphistry <> graphviz integration quickstart\n",
+        "\n",
+        "The [graphviz engine](https://graphviz.org/) is popular for layout of small graphs and rendering to static images. The Graphistry Python bindings to graphviz enable using pygraphistry as usual for quickly loading and manipulating your data, and then benefiting from graphviz for layout, and optionally, rendering.\n",
+        "\n",
+        "The example below shows laying out and rendering company ownership data that is in a tree and benefits from graphviz's high-quality layout engine."
+      ],
+      "metadata": {
+        "id": "fEjoJ5eBnuKZ"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Setup\n",
+        "\n",
+        "* graphviz: Install the graphviz engine and the pygraphviz bindings, see below (official [tutorial](https://pygraphviz.github.io/documentation/stable/install.html) )\n",
+        "* Graphistry: Install PyGraphistry below, and [get a free GPU account on Graphistry Hub](https://www.graphistry.com/get-started) or run your own server\n",
+        "\n",
+        "Notes:\n",
+        "\n",
+        "* You must install the graphviz engine, as well as its pygraphviz Python bindings and pygraphistry\n",
+        "* graphviz is most known for its `\"dot\"` layout engine, and it includes others as well\n",
+        "* graphviz is generally not recommended for layout of graphs over 10,000 nodes and edges"
+      ],
+      "metadata": {
+        "id": "0BF6FBhDpLas"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#!sudo apt-get install graphviz graphviz-dev\n",
```

Comment:
running sudo in a notebook doesn't really work. could either make it clear this needs to be run in a normal terminal or do something like https://stackoverflow.com/questions/44996933/using-sudo-inside-jupyter-notebooks-cell

Reply:
ah yeah tx will remove